src/Response.php

```diff
@@ -99,7 +99,7 @@ public function toResponse($request)
         $page = [
             'component' => $this->component,
             'props' => $props,
-            'url' => $request->getBaseUrl().$request->getRequestUri(),
+            'url' => $request->getBaseUrl().$request->getPathInfo(),
```

Comment:
fullUrl seems to work on my initial tests, but also adds scheme/protocol and host to the url. This would break some of the other tests. The current change in this pull request would technically be a smaller change, but fullUrl might be better, but unsure if there are other consequences except for broken tests.

Reply:
I changed to `'url' => $request->getRequestUri(),`  but I will test it sooner.